application/F3DStarter.cxx

```diff
@@ -8,7 +8,7 @@
 #include "F3DNSDelegate.h"
 #include "F3DOptionsTools.h"
 #include "F3DPluginsTools.h"
-#include "F3DSystemTools.h"
+#include "scene.h"
```

Comment:
put this include with other libf3d includes

Reply:
The scene.h was from the incorrect way of updating the scene, its been removed and I put the old include back since its more of a refactoring change and isn't relevant to this issue

Reply:
Undid change, was based on incorrect usage of internal api.